src/lib/features/listening-history/model/store.ts

```diff
@@ -31,12 +35,23 @@ const addToLocalListeningHistory = (podcastId: number) => {
 };
 
 export const addToListeningHistory = async (episode: Episode) => {
```

Comment:
Seeing as there is no `await` here and the function doesn't return any promises (or even values), why is it marked `async`?

Reply:
I see that an async function (`addToCloudListeningHistory`) is called inside, but will the `try`/`catch` actually take effect without `await` or will it be a normal promise rejection?
Perhaps unit tests would be useful here

Reply:
I'll add unit tests (maybe) later

Reply:
Ok. And if you are interested in the answer to my other questions, [this SO answer](https://stackoverflow.com/questions/30649994/can-i-catch-an-error-from-async-without-using-await) says that un`await`ed promise rejections will **not** trigger the `catch` block.